### ISSUE TITLE
Demo code for train the model and the understanding

### ISSUE THREAD

**Issue by gwh0112** [MalongTech/CurriculumNet#3]
Could you provide the network training demo code? Or please help me with the following understanding in the paper:
Use the 3-subsets curriculum learning as example. 
1. In the first stage, train the network from scratch using subset-1. Learning rate starts from 0.1 and decrease, until converge. 
2. In the second stage, train the above converged model with subset 1 and 2. Do we still use the learning rate 0.1 as the starting point and wait for convergence?
3. In the last stage, repeat the step 2 with three subsets and weight for convergence. Learning rate the same (0.1 decrease)?
Thank you so much.

**Comment by Sisi0518** [comment]
I have the same confusion. I am confused about how the three-stage training is implemented. Is the model trained for 70*10^4 iterations in each stage with the lr schedule described in Section 3.4 (210*10^4 in total) of your paper, or are the three stages all contained in the 70*10^4 iterations? If it is the latter case, could you please tell me when are the noisy/high noisy data added to the training?

**Comment by mscottml** [comment]
The model is trained for 70*10^4 iterations in each stage with the lr schedule described in Section 3.4 of the paper (in total, it is 210*10^4).
